cadquery/cq.py

```diff
@@ -3451,19 +3452,43 @@ def cutBlind(
         """
         # Handling of `until` passed values
         s: Union[Compound, Solid, Shape]
+        if isinstance(both, float) and taper == None:
```

Comment:
Should the deprecation warning include a check for `int` type input?

Reply:
@lorenzncode `int` will block boolean.

```
>>> isinstance(True, int)
True
```

Reply:
The pytest also required an explicit conversion to float to trigger the warning which may not be typical of normal usage.